app/rest/model/src/main/java/io/syndesis/model/integration/IntegrationDeployment.java

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.model.integration;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.syndesis.core.SuppressFBWarnings;
+import io.syndesis.model.Kind;
+import io.syndesis.model.WithId;
+import io.syndesis.model.WithKind;
+import io.syndesis.model.WithName;
+import io.syndesis.model.action.ConnectorAction;
+
+import java.math.BigInteger;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@JsonDeserialize(builder = IntegrationDeployment.Builder.class)
+public class IntegrationDeployment implements WithKind, WithId<IntegrationDeployment>, WithName {
+
+    public static final String COMPOSITE_ID_SEPARATOR = ":";
+
+    private final Optional<String> id;
+    private final Optional<Integer> version;
+    private final Optional<String> integrationId;
+
+    private final String name;
+    private final Integer parentVersion;
```

Comment:
Wont parent version always be version-1? What's the value?

Reply:
My understanding is that we can modify any `deployment` in which case the parentVersion is going to be the version of the `deployment` modified.

Reply:
So I think the deployment is mutable, just not the spec of the integration that it holds.  The version I think tracks the version of the spec.  So I think this is safe to remove.